packages/design-system/src/lib/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { clsx, type ClassValue } from 'clsx'
+import { twMerge } from 'tailwind-merge'
+
+export function cn(...inputs: ClassValue[]) {
+  return twMerge(clsx(inputs))
```

Comment:
This seems like a very good idea 👍 , especially in the case of "conflicting styles" (according to https://stackoverflow.com/questions/69390216/how-to-properly-join-tailwind-css-classes-using-clsx). It would probably make sense to replace the corresponding use of tailwind-merge through this combined function in all components soon, but we probably need to be careful to sudden changes in appearance in the corresponding apps...